src/minimalin.c

```diff
@@ -72,6 +72,12 @@ static GPoint SOUTH_INFO_CENTER = { .x = 72, .y = 112 };
 static GPoint NORTH_INFO_CENTER = { .x = 72, .y = 56 };
 #endif
 
+#ifdef PBL_HEALTH
```

Comment:
Note that this ifdef doesn't seem to reflect whether a user has enabled or disabled the Pebble Health service, but rather if the platform has it. Still worth keeping this in for all OG pebble users, but can be removed if the config page includes a switch to toggle Health API integration.